src/components/Alerts/UserActionAlert.js

```diff
@@ -0,0 +1,22 @@
+import ExpenseAddedAlert from './ExpenseAddedAlert';
+import ExpenseDeletedAlert from './ExpenseDeletedAlert';
+
+const UserActionAlert = ({
+  userAction,
+  errorOccurred,
+  closeCallback,
+  expense,
+}) => {
+  const baseProps = { errorOccurred, closeCallback };
+
+  switch (userAction) {
```

Comment:
nice!

Reply:
interesting approach. let's see how it holds up over time.
maybe we should document which actions are available and what props they require.